content/blog/2023-03-28-why-does-ethereum-classic-have-value/index.md

```diff
@@ -0,0 +1,82 @@
+
+## ETC Is Programmable
+
+In addition to being sound money, the native cryptocurrency of ETC is programmable through smart contracts. 
+
+Smart contracts are software programs that application developers may write and when they send them to the network they become decentralized as they are replicated across all nodes in the system, making them decentralized applications or dapps, for short.
+
+This programmability, combined with its decentralization and the soundness of its money, gives ETC incredible value and versatility for users and developers alike.
+
+Smart contracts are a key for further trust minimization, making ETC even more valuable than simple ledger systems as Bitcoin and Litecoin, because decentralized applications are like autonomous agents that replace the need for service providers hosted in proprietary data centers or cloud services.
+
+## ETC Is Good Business For Miners
+
+As ETC is the largest proof of work smart contracts blockchain in the world, and has a market capitalization in the billions, it is a very good business for proof of work miners. 
+
+Because ETC still has a long way to run as it rises further to become one of the top 2 blockchains in the world, miners are flocking to the network for the opportunity to put to work their GPUs and ASIC machines, and many of them are accumulating ETC for the long term rather than liquidating it to pay their bills or to exchange for other assets.
+
+ETC is currently the fourth largest mining computing base after Bitcoin, DogeCoin, and Litecoin, but in the short term it will likely surpass DOGE and LTC to become the second largest mining base in the world.
+
+## ETC Is Good Business for Economic Nodes
+
+Economic nodes are those nodes in the blockchain that are operated by businesses that use ETC to generate revenues. Examples of these are exchanges, wallet providers, block explorers, node as a service (NaaS) providers, data services, crypto custody services, and many others.
+
+Historically, trading volume in ETC against fiat and other cryptocurrencies has ranked very high in exchanges. This makes ETC a very good business for these players as well as for software and hardware wallet providers.
+
+ETC is listed in most top exchanges and featured in most top wallet brands. As it is widely used by the public globally, it is featured in many data services and various blockchain access and custody providers.
+
+Because running an ETC node is simple and straightforward, many of these businesses are node operators who value the ability to verify the data, transactions, and blocks by themselves. This is the basis of the saying “run your own node!” in the industry.
+
+## ETC Is Good for Global Users
+
+ETC is incredibly valuable for users globally. 
+
+Because it is a proof of work blockchain, with a fixed monetary policy, and smart contracts, it provides something that the world will increasingly demand as time passes by: Unstoppability.
```

Comment:
Line 70: Add hyphens; Remove unneeded commas; Replace pronoun; Remove redundant preposition
```suggestion
Because it is a proof-of-work blockchain with a fixed monetary policy and smart contracts, ETC provides something that the world will increasingly demand as time passes: unstoppability.
```